portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,58 +29,46 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Date;
-
-
- 
+import com.google.gson.Gson; 
 
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
 	private List<Comment> comments;
 
- 	@Override
-    public void init() {
-        comments = new ArrayList<>();
-    }
-
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
-        Query query = new Query("Comment").addSort("date", SortDirection.DESCENDING);
+        String commentNumStr = getParameter(request, "number", "5");
```

Comment:
I think it is a good idea to document the default number of `5`.

Reply:
You can store the default number as a static constant, with this style guidance: https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names